sdc/sdc_autogenerated.py

```diff
@@ -44,40 +44,46 @@
 from sdc.datatypes.common_functions import (sdc_join_series_indexes, sdc_check_indexes_equal)
 from sdc.hiframes.pd_series_type import SeriesType
 from sdc.str_arr_ext import (string_array_type, str_arr_is_na)
-from sdc.utilities.utils import sdc_overload
-from sdc.functions.numpy_like import astype
+from sdc.utilities.utils import sdc_overload, sdc_overload_method
+from sdc.functions import numpy_like
 
 
-@sdc_overload(operator.add)
-def sdc_pandas_series_operator_add(self, other):
+@sdc_overload_method(SeriesType, 'add')
+def sdc_pandas_series_add(self, other, level=None, fill_value=None, axis=0):
     """
-    Pandas Series operator :attr:`pandas.Series.add` implementation
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
 
-    Note: Currently implemented for numeric Series only.
-        Differs from Pandas in returning Series with fixed dtype :obj:`float64`
+    Pandas API: pandas.Series.add
 
-    .. only:: developer
+    Limitations
+    -----------
+    Parameters ``level`` and ``axis`` are currently unsupported by Intel Scalable Dataframe Compiler
 
-    **Test**: python -m sdc.runtests -k sdc.tests.test_series.TestSeries.test_series_op1*
-              python -m sdc.runtests -k sdc.tests.test_series.TestSeries.test_series_op2*
-              python -m sdc.runtests -k sdc.tests.test_series.TestSeries.test_series_operator_add*
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series/series_add.py
+       :language: python
+       :lines: 27-
+       :caption: Getting the addition of Series and other
+       :name: ex_series_add
 
-    Parameters
-    ----------
-    series: :obj:`pandas.Series`
-        Input series
-    other: :obj:`pandas.Series` or :obj:`scalar`
-        Series or scalar value to be used as a second argument of binary operation
+    .. command-output:: python ./series/series_add.py
+       :cwd: ../../../examples
 
-    Returns
-    -------
-    :obj:`pandas.Series`
-        The result of the operation
-    """
+    .. seealso::
 
-    _func_name = 'Operator add().'
```

Comment:
_func_name will not be defined in the function, and it's used in error-handling branches.